packages/calcite-components/package.json

```diff
@@ -71,6 +71,7 @@
     "util:hydration-styles": "tsx support/hydrationStyles.ts",
     "util:is-working-tree-clean": "[ -z \"$(git status --porcelain=v1)\" ]",
     "util:prep-build-reqs": "npm run util:copy-assets",
+    "util:sync-t9n-en-bundles": "tsx support/syncEnT9nBundles.ts",
```

Comment:
Sidebar: should we add this as pre-commit hook instead of a npm script?

Reply:
> Were you thinking we would run it on commit if there was a staged main translation bundle?

Yuss!

Reply:
Done! I'll proceed to install, but please give it a try and LMK if this needs any further tweaks.